ui/lib/core/addon/components/key-value-header.js

```diff
@@ -16,9 +16,14 @@ import { encodePath } from 'vault/utils/path-encoding-helpers';
  * @param {string} [root=null] - Used to set the secretPath.
  * @param {boolean} [showCurrent=true] - Boolean to show the second part of the breadcrumb, ex: the secret's name.
  * @param {boolean} [linkToPaths=true] - If true link to the path.
+ * @param {boolean} [isEngine=false] - Change the LinkTo if the path is coming from an engine.
  */
 
 export default class KeyValueHeader extends Component {
+  get isEngine() {
```

Comment:
I go back and forth on this. But because by default I believe this will be false if undefined, should I even do a default here for isEngine? Thoughts?

Reply:
Looks like I need it defined as false for some of the conditionals to work. Undefined on the template helpers isn't enough in the cases of #if to trigger a false 🤷‍♀️